build.gradle

```diff
@@ -101,13 +101,17 @@ ext.jmxetricsVersion = "1.0.8"
 ext.hadoopVersion = "2.7.2"
 
 dependencies {
-  compile ("org.apache.hadoop:hadoop-client:${hadoopVersion}") {
+
+  /* Note this jar is fetched from our private Maven repo (patched for HADOOP-12807) */
+  compile ("org.apache.hadoop:hadoop-aws:${hadoopVersion}") {
+    exclude group: "com.sun.jersey"
+  }
+
+  compile ("org.apache.hadoop:hadoop-common:${hadoopVersion}") {
```

Comment:
Because `haoop-aws` also depends on `hadoop-common` which depends on `com.sun.jersey`. Putting `hadoop-client` there is not enough to kick 1.9 away.